src/render/SDL_yuv_sw.c

```diff
@@ -26,6 +26,48 @@
 
 #include "SDL_yuv_sw_c.h"
 
+/*
+ * Output 'pitch' that can be used with SDL_ConvertPixels()
+ */
+int SDL_SW_SizeYUV(Uint32 format, int w, int h, int *pitch)
```

Comment:
Can you move this to vide/SDL_yuv_c.h and rename it to SDL_CalculateYUVSize()?

Reply:
done